app/namespaces/data_provider/test_data_provider_endpoints.py

```diff
@@ -0,0 +1,107 @@
+from app import db as _db
```

Comment:
OVERALL COMMENT: I don't like that all these files have been taken out of the /tests folder. It's best practice IMO to keep the testing stuff in a test folder. Can we plz move it back to the original structure?

Reply:
I actually kind of agree with Rahul that having tests in the same directory as the codebase being tested will make people more likely to update tests, so I'm not entirely in favour of this.

@simonarocco @rahularoradfs